lib/fake_stripe/fixtures/retrieve_charge.json

```diff
@@ -1,53 +1,88 @@
-    "object": "card",
-    "last4": "4242",
-    "dynamic_last4": null,
-    "brand": "Visa",
-    "exp_month": 1,
-    "exp_year": 2050,
-    "fingerprint": "ZYXWVUTSRQPONMLK",
-    "customer": null,
-    "country": "US",
-    "name": null,
-    "address_line1": null,
-    "address_line2": null,
-    "address_city": null,
-    "address_state": null,
-    "address_zip": null,
-    "address_country": null,
-    "cvc_check": "pass",
-    "address_line1_check": null,
-    "address_zip_check": "pass"
+  "amount": 900,
+  "amount_captured": 900,
+  "amount_refunded": 0,
+  "application": null,
+  "application_fee": null,
+  "application_fee_amount": null,
+  "balance_transaction": "txn_25i3ndwKVAMbOG",
+  "billing_details": {
+    "address": {
+      "city": null,
+      "country": null,
+      "line1": null,
+      "line2": null,
+      "postal_code": null,
+      "state": null
+    },
+    "email": null,
+    "name": "Silver",
+    "phone": null
   },
+  "calculated_statement_descriptor": null,
   "captured": true,
-  "refunds": [
-
-  ],
+  "created": 1372305853,
+  "currency": "usd",
+  "customer": "cus_25i3PhOUEUzwvs",
+  "description": null,
+  "disputed": false,
+  "failure_code": null,
+  "failure_message": null,
+  "fraud_details": {},
+  "invoice": "in_25i3zvFulsogvB",
+  "livemode": false,
+  "metadata": {},
+  "on_behalf_of": null,
+  "order": null,
   "outcome": {
-    "network_status": "approved_by_network",
+    "network_status": null,
     "reason": null,
-    "risk_level": "normal",
+    "risk_level": "not_assessed",
     "seller_message": "Payment complete.",
     "type": "authorized"
   },
-  "balance_transaction": "txn_103cQg2eZvKYlo2CYKVlR2nh",
-  "failure_message": null,
-  "failure_code": null,
-  "amount_refunded": 0,
-  "customer": null,
-  "invoice": null,
-  "description": null,
-  "dispute": null,
-  "metadata": {
-  }
+  "paid": true,
+  "payment_intent": null,
+  "payment_method": "cc_25i39sSFqHf3F7",
+  "payment_method_details": {
+    "card": {
+      "brand": "visa",
+      "checks": {
+        "address_line1_check": null,
+        "address_postal_code_check": null,
+        "cvc_check": "pass"
+      },
+      "country": "US",
+      "exp_month": 1,
+      "exp_year": 2018,
+      "fingerprint": "gKmAwmokVhNI2Yzr",
+      "funding": "credit",
+      "installments": null,
+      "last4": "4242",
+      "network": "visa",
+      "three_d_secure": null,
+      "wallet": null
+    },
+    "type": "card"
+  },
+  "receipt_email": null,
+  "receipt_number": null,
+  "receipt_url": "https://pay.stripe.com/receipts/acct_1wuZLSIRFAiR8lgNnDWn/ch_25i3MyhoGLLA8s/rcpt_EH70VCU16gj9NofxyLId16TVfaFR65I",
+  "refunded": false,
+  "refunds": {
+    "object": "list",
+    "data": [],
+    "has_more": false,
+    "url": "/v1/charges/ch_25i3MyhoGLLA8s/refunds"
+  },
+  "review": null,
+  "shipping": null,
+  "source_transfer": null,
+  "statement_descriptor": null,
+  "statement_descriptor_suffix": null,
+  "status": "succeeded",
```

Comment:
For some reason this fixture doesn't have `status` on it?  Strangely enough the latest from [Thoughbot](https://github.com/thoughtbot/fake_stripe/blob/master/lib/fake_stripe/fixtures/retrieve_charge.json) doesn't either 🤔 🤷‍♂️  but I am going to go with what is on Stripes API and matches the JSON responses I get when I test it out on prod.